activerecord/lib/active_record/connection_adapters/abstract/quoting.rb

```diff
@@ -54,6 +55,10 @@ def quote_bound_value(value)
         quote(value)
       end
 
+      def quote_array(a)
```

Comment:
In don't think we want this to be part of the public API so we can move it under private or at least add nodoc.